app/views/content/salaries-and-benefits.md

```diff
@@ -52,11 +52,13 @@ keywords:
 
 For helping to shape the next generation, you’re entitled to a competitive salary, generous holidays, and a substantial pension.
 
-Figures are for the 2022/23 academic year.
+Figures are for the 2022/23 academic year and apply to maintained primary, secondary and special schools in England (schools funded by local authorities).
```

Comment:
@EllieNodder Is it worth bolding `Figures are for the 2022/23 academic year` so it doesn't get lost? But if this is a content no-no, happy as is!

Reply:
hmm we don't really use bold to emphasise text in regular content pages (and the gov.uk style guide says to use it sparingly) so would be inclined not to do so here

Reply:
Is it worth splitting into 2 sentences? 

```suggestion
Figures are for the 2022/23 academic year. 

These salaries apply at maintained primary, secondary and special schools in England (schools funded by local authorities).
```